src/components/singleShoppingList.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Checkbox,
+  Dialog,
+  DialogTitle,
+  ListItem,
+  List,
+  ListItemIcon,
+  ListItemText,
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  root: {
+    height: 800,
+    width: 2000,
+  },
+});
+
+const SingleShoppingList: React.FC<SingleShoppingList> = ({
+  open,
+  onClose,
+}) => {
+  const classes = useStyles();
+  const [checked, setChecked] = React.useState([0]);
+  const handleToggle = (value: number) => () => {
+    const currentIndex = checked.indexOf(value);
+    const newChecked = [...checked];
+
+    if (currentIndex === -1) {
+      newChecked.push(value);
+    } else {
+      newChecked.splice(currentIndex, 1);
+    }
+
+    setChecked(newChecked);
+  };
+  const shoppingListItems = [
+    { item: 'item 1', id: 1 },
+    { item: 'item 2', id: 2 },
+    { item: 'item 3', id: 3 },
+    { item: 'item 4', id: 4 },
+    { item: 'item 5', id: 5 },
+    { item: 'item 6', id: 6 },
+    { item: 'item 7', id: 7 },
+    { item: 'item 8', id: 8 },
+    { item: 'item 9', id: 9 },
+  ];
+
+  return (
+    <Dialog
+      onClose={onClose}
+      aria-labelledby="simple-dialog-title"
+      open={open}
+      PaperProps={{ classes: { root: classes.root } }}
```

Comment:
it seems odd to me that this `PaperProps` prop is PascalCase (starts with uppercase). I guess it's fine if it's in the docs for material ui but ideally all our props start with lowercase